frontend/src/App.js

```diff
@@ -6,6 +6,9 @@ import ClientDataTable from "./ClientDataTable";
 import AddQuestionPage from "./AddQuestionPage";
 import LandingPage from "./LandingPage";
 import AdminPage from "./AdminPage";
+import SelectLanguage from "./SelectLanguage";
+import AdminPanel from "./AdminPanel";
```

Comment:
Remove `AdminPanel` import if we decide to not merge `AdminPanel.js`.
Refer to the comment on the file above.